### PR TITLE
[16.0][MIG] pos_customer_tree_view_vat: Migration to 16.0 

### PR DESCRIPTION
**Pull request by chienandalu**
Standard migration

cc @Tecnativa TT49799

please review @CarlosRoca13 @pilarvargas-tecnativa

### CODE REVIEW

File: pos_customer_tree_view_vat/static/src/xml/pos.xml [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
                 <t t-esc="props.partner.vat or ''" />
             </td>
         </xpath>
     </t>
-    <t t-inherit="point_of_sale.ClientListScreen" t-inherit-mode="extension" owl="1">
+    <t t-inherit="point_of_sale.PartnerListScreen" t-inherit-mode="extension" owl="1">
         <xpath expr="//thead/tr/th[1]" position="after">
             <th id="customer_tree_tax_label">Tax ID</th>
         </xpath>
     </t>
```
Comment (CarlosRoca13) [inline pos_customer_tree_view_vat/static/src/xml/pos.xml]: Set in separated files

File: pos_customer_tree_view_vat/__manifest__.py [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
     "website": "https://github.com/OCA/pos",
     "maintainers": ["mileo"],
     "depends": [
         "point_of_sale",
     ],
-    "qweb": [
-        "static/src/xml/pos.xml",
-    ],
+    "assets": {
+        "point_of_sale.assets": ["pos_customer_tree_view_vat/static/src/xml/pos.xml"]
```
Comment (CarlosRoca13) [inline pos_customer_tree_view_vat/__manifest__.py]:
```suggestion
        "point_of_sale.assets": [
            ("after", "point_of_sale/static/src/xml/Screens/PartnerListScreen/PartnerLine.xml", "pos_customer_tree_view_vat/static/src/xml/partner_line.xml"),
             ("after", "point_of_sale/static/src/xml/Screens/PartnerListScreen/PartnerListScreen.xml", "pos_customer_tree_view_vat/static/src/xml/partner_list_screen.xml"),
        ]

**Review by CarlosRoca13 — APPROVED** [review]
Just a non-blocking comment

I think that the extensions should be defined in separated files and charged just after the main module to avoid loosing it if some other module make a primary template of it.

**Comment by chienandalu** [comment]
@CarlosRoca13 changes done. Thanks for the tips :)

**Review by CarlosRoca13 — APPROVED** [review]
Thanks! :smile:

**Review by pilarvargas-tecnativa — COMMENTED** [review]
thanks! :+1:

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**